src/components/App/App.js

```diff
@@ -1,14 +1,19 @@
 
 import './App.css';
-import { Link } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
```

Comment:
This is super super nitpicky, but you have an empty line on line 1. My mentor also said it's better convention to have all of your dependencies come first in the imports (things like react, router, prop types) and then have less important things farther down like styles and other components.